### ISSUE TITLE
update & put Qubes OS roadmap on webite

### ISSUE THREAD

**Issue by mfc** [QubesOS/qubes-issues#2032]
Roadmap needs to be updated, and should be incorporated into Qubes OS website somehow:

https://github.com/rootkovska/qubes-roadmap

**Comment by andrewdavidwong** [comment]
CC @rootkovska

**Comment by mfc** [comment]
can we just copy this into QubesOS repo so that we can keep it updated?

**Comment by andrewdavidwong** [comment]
I'm not sure the location is really relevant. @rootkovska (or someone) would still have to update it, no matter which repo it resides in.

**Comment by mfc** [comment]
yes, then marek can merge requests to update it: https://github.com/rootkovska/qubes-roadmap/pull/3

**Comment by mfc** [comment]
I would like to refer to the roadmap in our funding blogpost, it needs to be updated at minimum please.

**Comment by mfc** [comment]
actually instead of putting the roadmap image on the website, I think it would be preferred to present the content as text on a page.

**Comment by mfc** [comment]
example of text-based, high-level project roadmap: https://tails.boum.org/contribute/roadmap/

**Comment by mfc** [comment]
removing help wanted since this is not something someone external to the project can do

**Comment by ben-grande** [comment]
@marmarek Should Github Projects be linked, instead of duplicating work to the website? It is not exactly a roadmap, but it is a low effort one for the next release, it shows what we are working on, which, I believe, is most important for users of immediate (1-2 years) needs.

**Comment by marmarta** [comment]
Maybe this is a good idea? I'm not 100% sure if projects aren't a bit messy...

**Comment by mfc** [comment]
> example of text-based, high-level project roadmap: https://tails.boum.org/contribute/roadmap/

i see tails removed their project roadmap & just have their gitlab milestones...

> Should Github Projects be linked, instead of duplicating work to the website? It is not exactly a roadmap, but it is a low effort one for the next release, it shows what we are working on, which, I believe, is most important for users of immediate (1-2 years) needs.

yes we could link to it somewhere, it is messy & very "in the weeds" but it is something concrete to view for someone interested. while i think a roadmap is higher level - like we are at R4.3, what are the changes that get us to R5.0 vs R4.4 - we dont have capacity to make/maintain that type of content. i will bump the priority of this down...
